Sources/BraintreeSEPADirectDebit/WebAuthenticationSession.swift

```diff
@@ -12,17 +12,19 @@ class WebAuthenticationSession: NSObject {
     func start(
         url: URL,
         context: ASWebAuthenticationPresentationContextProviding,
-        completion: @escaping (URL?, Error?) -> Void
+        sessionDidDisplay: @escaping (Bool) -> Void,
+        sessionDidComplete: @escaping (URL?, Error?) -> Void
     ) {
-        self.authenticationSession = ASWebAuthenticationSession(
+        let authenticationSession = ASWebAuthenticationSession(
             url: url,
             callbackURLScheme: BTCoreConstants.callbackURLScheme,
-            completionHandler: completion
+            completionHandler: sessionDidComplete
         )
 
-        authenticationSession?.prefersEphemeralWebBrowserSession = true
-        authenticationSession?.presentationContextProvider = context
-
-        authenticationSession?.start()
+        authenticationSession.prefersEphemeralWebBrowserSession = true
```

Comment:
I found something interesting, that this line suppresses alert button from showing up and from system notification of applicationDidBecomeActive. I am not sure why. 
But I am wondering if we need this for some European compliance and if so, we would not need to deal with alert view cancel.